docs/submission/dictionary-overview.md

```diff
@@ -5,24 +5,63 @@ title: Dictionary Overview
 
 The ICGC ARGO [Data Dictionary](/dictionary) expresses the details of the ARGO data model, which adheres to specific formats and restrictions to ensure a standard of data quality. The Data Dictionary defines a set of files, each related to a clinical concept, that can be submitted to the ARGO Data Platform.
 
-## Reading the Data Dictionary
+To follow the evolution of the data dictionary, check out the [Dictionary Release Notes] (../release-notes/dictionary-releases).
 
-// image of core tag and define core fields
+## Understanding the Data Dictionary Table View
 
-//image of id tag and define id fields
+The [dictionary table view](/dictionary) lists all of the clinical fields that the ARGO Data Platform accepts, separated by clinical TSV file.
 
-//image of dependency tag and define what a "dependency" is
+The first list is for the Sample Registration file, which is the only file to be uploaded in the **Register Samples** section. All other lists outline the fields in the clinical TSV files that will be uploaded in the **Submit Clinical Data** section. A list of links on the right side of the page makes it very simple to navigate the clinical files. Each file starts with the name, field count, description, and a file name example to help ensure proper names are used. 
 
-// explain how to use the regex checker (click the regex examples)
+The lists can be filtered by Data Tier and Attribute, which can help a data submitter identify which fields are necessary for [clinical data completion](clinical-data-validation-rules). 
 
-## List of Dictionary Reference Data bases
+The dictionary version appears in a dropdown above the lists, as well as a date of when it was last updated. You can explore previous dictionary versions by choosing another version number from the dropdown. This dictionary version will also be reflected in the names of the file templates that are downloaded from the Platform. It is important to use the most current files for the dictionary validation to run smoothly. 
```

Comment:
Should we say `clinical validation` instead of `dictionary validation`?

Reply:
yes thanks, I'll change that